package.json

```diff
@@ -16,7 +16,7 @@
     "sinon": "^1.17.6"
   },
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha tests/js/tests/*.test.js"
+    "test": "./node_modules/mocha/bin/mocha tests/js/tests/*.test.js --timeout 30000"
```

Comment:
I guess so. Just tested your "yii.captcha.test.js" locally in 31ms so everything seems to work fine.